src/providers/wms/qgswmssourceselect.cpp

```diff
@@ -291,6 +291,8 @@ bool QgsWMSSourceSelect::populateLayerList( const QgsWmsCapabilities &capabiliti
   const QVector<QgsWmsLayerProperty> layers = capabilities.supportedLayers();
   mLayerProperties = layers;
 
+  QString defaultEncoding = QgsSettings().value( "qgis/WMSDefaultFormat", "" ).toString();
```

Comment:
defaultFormat?

Reply:
Called it encoding just because it was referred as encoding below